.travis.yml

```diff
@@ -16,7 +16,7 @@ branches:
     - /^\d+\.\d+(\.\d+)?(-\S*)?$/
 
 env:
-  - BAZEL=0.5.4 TF=NIGHTLY
```

Comment:
We can keep this pinned to nightly and update the branch later, once we fix we fix TensorFlow.